config/locales/en_LK.yml

```diff
@@ -14,12 +14,11 @@ en_LK:
       facility_group:
         state: "Province"
       address:
-        street_address: "House and street number"
-        village_or_colony: "Village or colony"
-        zone: "Town *"
+        street_address: "House and road number"
+        village_or_colony: "Town/Area"
         district: "District"
-        state: "Province *"
-        country: "Country *"
+        state: "Province"
+        country: "Country"
```

Comment:
Our Ethiopia config _has_ the asterisks. Are we sure we want to remove them?

Reply:
We don't display the patient address fields anywhere on the server, and I was checking if we need to localise the model at all. We don't localise for India, Bangladesh for example. Do you think we can remove this localisation?

Reply:
Ah, I suspect then that this localization is used in patient line lists. Let's keep the localization, and remove the asterisks.